### PR TITLE
Saturday morning edits pass to clean things up

### PR DESCRIPTION
**Pull request by jchodera**
I'm just fixing minor problems at this point.

### CODE REVIEW

**Comment by jchodera** [comment]
@davidlmobley : It might be good to take a quick pass through the Specific Aims again, since I noticed Aim 3 was now out of date.

**Comment by jchodera** [comment]
I've finished my pass for now. Feel free to review and merge.

I only had time to incorporate a few of Mehtap's comments.

**Comment by jchodera** [comment]
Be sure to see the email I sent last night with my updated Biosketch. The wrong biosketch somehow ended up in the proposal (probably the one from the github repo, not the one I emailed you).

File: latex/Research_Strategy.tex [hunk 1]
```diff
@@ -243,29 +244,32 @@
...
-Data collection/curation is followed by a submission window for that challenge component, then all results and analysis are returned to participants and posted on the SAMPL website; this also will nucleate more detailed discussion on the relevant Slack channel (which is expected to already be underway as participants prepare during the data collection phase). 
+Data collection/curation is followed by a submission window for that challenge component, then all results and analysis are returned to participants and posted on the SAMPL website; this also will nucleate more detailed long-term discussion on the relevant Slack channel.
 Each component will then wrap up with a focused virtual meeting where participants focus on lessons learned and areas which need further exploration; these will be recorded and posted on our website to assist in rapid dissemination of new insights. 
 Virtual meetings precede the submission window for the next SAMPL challenge, giving the opportunity to incorporate lessons learned for the next challenge.
+Release of curated datasets as high-quality benchmark sets will immediately follow data unblinding.
```
Comment (davidlmobley) [inline latex/Research_Strategy.tex:258]: I'll tweak this slightly to something more concise, perhaps "Immediately following the submission window, curated datasets will be released as high quality benchmarks."

Comment (jchodera) [reply latex/Research_Strategy.tex:258]: Perfect. I just wanted to make sure we included some mention of this.

**Pull request merged**